canonical-web-and-design/ubuntu.com
# Restructure the docs to use Vanilla docs layout

## Done
- Build two central docs templates (discourse or markdown) with Vanilla docs layout
- Remove all the other docs templates
- Set all docs pages to use the new templates
- Update the docs to use `h3` instead of `strong` titles in the navigation
- As this changes the main content from a 12 column layout to a 9 column I had to change a few classes in the markdown
- Removed the old docs layout styling
- Made the ceph navigation persistent as you navigate the subpages

## QA
- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- There are four docs (2 from discourse and 2 as markdown). Please check through the following sections:
  - /kubernetes/docs (markdown)
  - smartstart/guide (markdown)
  - server/docs (discourse)
  - ceph/docs (discourse)

## Issue / Card
Fixes https://github.com/canonical-web-and-design/ubuntu.com/issues/8856
Fixes #8874
